content/docs/getting-started/user-management/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: User Managment in Backstage
+lastUpdated: '2021-09-21T14:00:00.0Z'
+description: How to manage users in Backstage
+---
+
+## Introduction
+
+Roadie provides backstage with an authentication and user management feature for your new tenant.
+
+When the tenant is created the user requesting the tenant is added as the first user in backstage. This user may choose to add another user.
+
+## User Management
+The user management page is contained under the Administration side bar menu item. ![admin-sidebar.png](admin-sidebar.png)
+
+![user-management-page.png](user-management-page.png)
+
+## Creating a User
+To add a new user click the invite a new user button and enter the users email address.
+
+![invite-user-button.png](invite-user-button.png)
+
+Enter the users email and click send.
+
+![add-user-dialog.png](add-user-dialog.png)
+
+We will send an email to the users email address. The user will be asked to set a password and will them be redirected to login to backstage.
+
+## Removing a User
+At present we do not provide the ability to remove a user. However, we can take care of that on your behalf. Please speak to a support person and we will do that for you.
+
+## Custom Authentication Provider
```

Comment:
In my experience, most people call this "Single Sign On". Having that term in the heading will make this easier to find.

```suggestion
## Single Sign On
```